UnityClient/Assets/Scripts/TowerDefence/Enemy.cs

```diff
@@ -5,10 +5,14 @@
 
 public class Enemy : MonoBehaviour
 {
+    public float startSpeed = 10f;
+
+    [HideInInspector] public float speed;
+    
     public float startHealth = 100;
     private float health;
```

Comment:
Please update the naming convention of this to `_health` to bring it in line with the rest of the codebase.